hack/manifests/csi/vsphere/04-example-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: example-windows-pvc
+  namespace: windows-webserver
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
```

Comment:
I recommend going with something smaller as it may lower the odds of the datastore running out of storage capacity.

Reply:
Changed to 2Gi